src/support/z_module_manager.erl

```diff
@@ -152,6 +160,12 @@ activate(Module, IsSync, Context) ->
                         end
                 end,
             1 = z_db:transaction(F, Context),
+            UId = z_acl:user(Context),
+            z:warning(
```

Comment:
Again, these should not be warnings IMHO.

Reply:
See also above, not really a "warning" as such, more an high priority informational message.